tests/Unit/Emogrifier/HtmlProcessor/CssInlinerTest.php

```diff
@@ -1377,7 +1409,11 @@ public function mediaRulesDataProvider()
             'style in "screen" media type rule' => ['@media screen {p {color: #000;}}'],
             'style in "print" media type rule' => ['@media print {p {color: #000;}}'],
             'style in "all" media type rule' => ['@media all {p {color: #000;}}'],
-        ];
+        ] + $this->getDatasetsWithVaryingWhitespace([
+            'style in media type rule' => [' @media { p { color : #000; } } '],
+            'style in "screen" media type rule' => [' @media screen { p { color : #000; } } '],
+            'style in "only screen" media type rule' => [' @media only screen { p { color : #000; } } '],
```

Comment:
The same as above - please use string literals here to make this more straightforward to read.